src/core/internal/convert.d

```diff
@@ -314,21 +354,49 @@ private uint binLog2(T)(const T x)
 }
 
 @safe pure nothrow @nogc
-private ulong denormalizedMantissa(T)(T x) if(floatFormat!T == FloatFormat.Real80)
+private Float denormalizedMantissa(T)(T x, uint sign) if(floatFormat!T == FloatFormat.Real80)
```

Comment:
I don't suppose these could be named `floatDenormalizedMantissa` or `floatWithDenormalizedMantissa` to make it more clear from the caller what they return.

Not too fussed either way.

Reply:
Private function only used in this module at CTFE, don't think it's worth kicking off the CI again for this trivial rename.